gradle.properties

```diff
@@ -9,7 +9,7 @@ junit5Version=5.8.2
 junitPlatformVersion=1.8.2
 googleTruthVersion=1.1
 
-aaKotlinBaseVersion=1.9.30-dev-1098
+aaKotlinBaseVersion=1.9.30-dev-2548
```

Comment:
Can you reword the commit message, so that it can be auto-merged?

Reply:
can you elaborate? all 3 commits have same commit message, I tried to run the shell command locally and can have all 3 commits filtered.

Reply:
The commit for AA should be cherry-picked automatically. With `UPDATE_KOTLIN_VERSION` in commit message it will be filtered out.

Reply:
unfortunately due to the dependency issues in `common-util` mentioned in the description, AA commit has to be excluded from auto-merge.

Reply:
KSPAATest in the release branch is redundant. Let's disable it.

Talking about tests, we need to enable KSP 2 in gradle tests and integration tests. Could you update CI to include KSP 2.0 in the matrix in a separate PR?